src/documentation/common.md

```diff
@@ -58,4 +58,75 @@ The files `contributors.txt` and `datasets.txt` provide additional information a
 | datasets.txt | dataset_id | dataset_id | Required | This field is prefixed.
 | datasets.txt | contributor_id | contributor_id | Required | This field is prefixed.
 | datasets.txt | dataset_start_date |  |  | Smallest date of all the trips of the dataset.
-| datasets.txt | dataset_end_date |  |  | Greatest date of all the trips of the dataset.
\ No newline at end of file
+| datasets.txt | dataset_end_date |  |  | Greatest date of all the trips of the dataset.
+
+## Common practices
+The following rules apply to every converter, unless otherwise explicitly specified.
+- When one or more stop_points in the input data are not attached to a stop_area, 
+a stop_area is automatically created. The coordinates of the new stop_area are 
+computed as the barycenter of the associated stop_points. (The same rule applies 
+in case the stop_area exists in the input data, but the coordinates are not specified.)
+- Unless otherwise specified, dates of service are trasnformed into a list of active dates as if using a single NTFS file `calendar_dates.txt`.
+
+| NTFS file | NTFS field | Constraint | Note |
+| --- | --- | --- | --- |
+| calendar_dates.txt | service_id | Required | All slashes `/` are removed
```

Comment:
are you sure all slashes are removed?
I know that slashes are removed in trip, stop time, stop in gtfs connector.

Reply:
Actually, it's interesting to note that slashes are only removed in the GTFS2NTFS connector. So to be precise, slashes are removed only on `stop.id` and `shape.id` only on the read of a GTFS file. Nothing is done (at least, from what I can see) on the NTFS side. That said, that might be considered a bug... not sure.

Reply:
the problem has only been encounter with gtfs comming from osm (osm2gtfs tool). IMHO it's a good practice to centralize this kind of "can beak navitia api" data errors.